local-tests/tests/testUseEoaSessionSigsToEncryptDecryptFile.ts

```diff
@@ -25,13 +25,9 @@ export const testUseEoaSessionSigsToEncryptDecryptFile = async (
     userAddress: alice.wallet.address,
   });
 
-  const eoaSessionSigs = await getEoaSessionSigs(devEnv, alice);
-
   const encryptRes = await LitJsSdk.encryptString(
     {
       accessControlConditions: accs,
-      chain: 'ethereum',
```

Comment:
Chain is still required or was that never being used for encryption?

Reply:
yea, i digged into the code and it's not being used anymore, i guess it was used for storing on chain